airflow/config_templates/default_airflow.cfg

```diff
@@ -790,3 +790,8 @@ fs_group =
 # The worker pods will be given these static labels, as well as some additional dynamic labels
 # to identify the task.
 # Should be supplied in the format: key = value
+
+[analytics]
```

Comment:
I wonder if this should live under the webserver section as `analytics_tool`, and `analytics_id`? (It _only_ applies to the webserver.)

At the very least we should probably place this in the file directly after the `[webserver]` section if we keep the name as this.

Reply:
A separate section name makes sense (if we wanted to add support for pwik/matomo then we'd want a server/host too, so having it's own section makes this easier), but lets move it in the file.